graspy/version/version.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import datetime
+from typing import List
+import pkg_resources
+
+__all__: List[str] = ["version", "name"]
+
+name = "graspy"  # TODO: #454 Update in https://github.com/microsoft/graspologic/issues/454
+
+# manually updated
+__semver = "0.1.0"
+#  full version (may be same as __semver on release)
+__version_file = "version.txt"
```

Comment:
Need `recursive-include gaspologic/version version.txt` included in `MANIFEST.in` since this files not included in installation. I know `package_data` in setup.py is supposed to handle it, but using `package_data` never really worked for me